pages/common/funzip.md

```diff
@@ -0,0 +1,12 @@
+# funzip
+
+> Print the content of the first (non-directory) member in an archive without extraction.
+> More information: <https://manned.org/funzip>.
+
+- Print the content of the first member in a `.zip` archive:
+
+`funzip {{path/to/archive}}`
+
+- Decrypt a `.gz` archive and print the content:
```

Comment:
The `-password` doesn't work with Zip?

Reply:
Should we mentioned `zip` and `gz` in description at all location? the command are same. the only thing that change is extrension.

Reply:
my bad, it make sense to add `zip` and `gz` is 2nd example only